routing/turns_generator.cpp

```diff
@@ -668,6 +668,8 @@ size_t CheckUTurnOnRoute(TUnpackedPathSegments const & segments,
   auto const & masterSegment = segments[currentSegment - 1];
   if (masterSegment.m_path.size() < 2)
     return 0;
+  if (segments[currentSegment].m_path.empty())
```

Comment:
Small note: CheckUTurnOnRoute - does it return 1-based index of segment and 0 is some non-invalid value? Would be good to have short description about the function.